plugins/antispam/antispam_plugin.cpp

```diff
@@ -95,10 +134,10 @@ PCL void OnInfoRequest(pluginInfo_t *info){	// Function used to obtain informati
     info->handlerVersion.minor = PLUGIN_HANDLER_VERSION_MINOR;	// Requested handler version
 
     // =====  OPTIONAL  FIELDS  =====
-    info->pluginVersion.major = 1;
-    info->pluginVersion.minor = 0;	// Plugin version
-    strncpy(info->fullName,"IceOps antispam plugin by TheKelm",sizeof(info->fullName)); //Full plugin name
-    strncpy(info->shortDescription,"This plugin is used to prevent spam in the ingame chat.",sizeof(info->shortDescription)); // Short plugin description
-    strncpy(info->longDescription,"This plugin is used to prevent spam in the ingame chat. To personalize the settings, set corresponding cvars. Copyright (c) 2013 IceOps",sizeof(info->longDescription));
+    info->pluginVersion.major = 2;
+    info->pluginVersion.minor = 1;	// Plugin version
+    strncpy(info->fullName,"Cod4X Antispam plugin by Usmania",sizeof(info->fullName)); //Full plugin name
```

Comment:
the plugin was not created by you. editing a handful of lines doesnt make you the creator.

Reply:
Oh yeah my bad. I rewrote the whole thing just to get the hang of the logic and different Functions.
I will change it back.